src/Launcher/LauncherBase.php

```diff
@@ -184,9 +132,9 @@ public function launchJobs($jobs) {
   }
 
   /**
-   * Format running state.
+   * {@inheritdoc}
    */
-  public function formatRunning($job) {
+  public function formatRunning(CronJob $job) {
```

Comment:
There are many methods here that we are not yet really calling. See the list_build_row() method in the list builder, that's the old code for the overview. We should consider calling these methods in a similar way and have the same information there. Another task..